### PR TITLE
Changes after code review by Mariotto and Cestaro

### PR DESCRIPTION
**Pull request by turcato-niccolo**
https://github.com/turcato-niccolo/SmsHandler/pull/2/

(Fixed mistake in KadAction due to change of interface for SMSPeer)

### CODE REVIEW

File: kademlia/src/main/java/ingsw/group1/kademlia/KadActionsBuilder.java [hunk 1]
```diff
@@ -28,7 +26,7 @@
      */
     public KadActionsBuilder(int maxActionID) {
         if (maxActionID >= KadAction.MIN_ID && maxActionID <= KadAction.MAX_ID)
-            MAX_ACTION_ID = maxActionID;
+            maxActionId = maxActionID;
```
Comment (RicDeZen) [inline kademlia/src/main/java/ingsw/group1/kademlia/KadActionsBuilder.java]: Codacy Issue found: [Possible unsafe assignment to a non-final static field in a constructor.](https://app.codacy.com/manual/RicDeZen/ingsw-group1-kademlia/pullRequest?prid=4761148)

**Comment by RicDeZen** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
- Added 2
           

Complexity decreasing per file
==============================
+ kademlia/src/main/java/ingsw/group1/kademlia/BitSetUtils.java  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/RicDeZen/ingsw-group1-kademlia/pullRequest?prid=4761148&bid=15876064)

File: kademlia/src/main/java/ingsw/group1/kademlia/KadActionsBuilder.java [hunk 2]
```diff
@@ -38,17 +36,17 @@ public KadActionsBuilder(int maxActionID) {
      * Default constructor, builder will accept all valid IDs, (as indicated in KadAction)
      */
     public KadActionsBuilder() {
-        MAX_ACTION_ID = KadAction.MAX_ID;
+        maxActionId = KadAction.MAX_ID;
```
Comment (RicDeZen) [inline kademlia/src/main/java/ingsw/group1/kademlia/KadActionsBuilder.java]: Codacy Issue found: [Possible unsafe assignment to a non-final static field in a constructor.](https://app.codacy.com/manual/RicDeZen/ingsw-group1-kademlia/pullRequest?prid=4761148)

**Comment by ricdezen** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
           

Complexity decreasing per file
==============================
+ kademlia/src/main/java/ingsw/group1/kademlia/BitSetUtils.java  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/RicDeZen/ingsw-group1-kademlia/pullRequest?prid=4761148&bid=15876064)

**Review by ricdezen — APPROVED** [review]

**Pull request merged**